resources/assets/js/utilities/VoterRegistrationContent.js

```diff
@@ -0,0 +1,28 @@
+const $ = require('jquery');
+
+/**
+ * Utility script to enable toggling visibility of voter registration content
```

Comment:
Similar to above re the naming, can we amend this to: 
```suggestion
 * Utility script to enable toggling visibility of voter registration call to action on '/profile/about' form
```